src/jarabe/journal/listview.py

```diff
@@ -747,7 +747,11 @@ def __icon_clicked_cb(self, cell, path):
 
     def __cell_title_edited_cb(self, cell, path, new_text):
         iterator = self._model.get_iter(path)
-        self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+        if new_text.isspace() or new_text == '':
+            return
+        else:
+            self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+
         self.emit('title-edit-finished')
```

Comment:
Yeah, This is a better idea.
@samdroid-apps @godiard I have made the changes. Have a look